### ISSUE TITLE
No Attachments

### ISSUE THREAD

**Issue by wfoster** [osTicket/osTicket-1.8#2355]
Hi Guys

I have an issue with attachments which fail to import into the system.
The system is being used via email. So basically email attachments are sometimes not coming in.
Is there a quick fix for this?

Thanks

**Comment by greezybacon** [comment]
Yes. Quit using attachments. That's the easiest "quick fix"

Am Sonntag, 2. August 2015 schrieb wfoster :

**Comment by wfoster** [comment]
Unfortunately that is not an option for us. We have guys who need to email in screenshots of errors.
It works 90% of the time. I just cannot understand why it does not work that random 10%.

**Comment by greezybacon** [comment]
I'll need some more information to find and fix a handling issue with
attachments.

Perhaps you could start with versions of osTicket, Apache or IIS, MySQL
you're using, if you're piping or fetching email, if fetching, if you're
using POP or IMAP, what are the attacent settings you have in place?, are
there any errors in your log in osTicket or php error log?, are there any
internal notes in the tickets with dropped attachments describing an import
issue?, can you supply a raw email (with headers) of a mail that is
resulting in dropped attachments.

Perhaps also helpful, what mail system is brokering your mail (eg
Exchange), and does an email client in particular aggregate the problem (eg
Android Mobile).

Cheers,
Jared

Am Sonntag, 2. August 2015 schrieb wfoster :

**Comment by wfoster** [comment]
Hi Jared

I went into an investigation regarding this and discovered that all the errors are from guys who have attached emails to their replies. So I have instructed the staff to le the users know that they must attach the files itself.

Thanks for all of the help.
Will log on and close the issue



Wesley Foster | Network Administrator
Tel: +27 21 506 4496 | Fax: +27 21 511 4256 | Cell: +27 71 344 0339
Email: wesleyf@novagroup.co.za<mailto:wesleyf@novagroup.co.za> | Web: www.novagroup.co.za<http://www.novagroup.co.za/>
